<launch>
    <!-- 启动 YOLOv8 检测节点 -->
    <node name="yolov8_detector" pkg="yolov8_ros" type="detector.py" output="screen">
        <param name="model_path" value="$(find yolov8_ros)/models/best.pt" />
    </node>
</launch>
